### ISSUE TITLE
Coffeescript formatting broken

### ISSUE THREAD

**Issue by alkama** [Glavin001/atom-beautify#168]
First, lets say all versions are up to date, both for atom and atom-beautify.

**Here is a strange behavior I see:**
* When trying to beautify a CoffeeScript file, I get a ```undefined is not a function``` message in the "Atom Beautify Error Messages" MessagePanelView.

I tried to debug it and it fails here: https://github.com/Glavin001/atom-beautify/blob/master/lib/langs/coffeescript-beautify.coffee#L11 

It made me think the dependant module was probably wrongly installed, and indeed something gets messed !

**Here are some details:**
* When looking at the ```atom-beautify/node_modules/coffee-formatter``` folder (so after a successful ```apm install```) I see that structure:
```
README.md
node_modules/
package.json
```
Something is obviously missing there... the library itself !

* When looking at the ```atom-beautify/package.json``` file (which is enriched with caches by the successful apm installation) I see even more puzzling insight:
```
"_atomModuleCache": {
  "version": 1,
  "dependencies": [
  ...
  {
    "name": "coffee-formatter",
    "version": "0.1.0",
    "path": "node_modules/atom-message-panel/lib/index.js"
  },
  ...
```
Notice the crazy "path" unfolding to a completely wrong location.

I guess there is something missing in the current state of the "coffee-formatter" plugin.
I saw that it was written in litcoffee, with grunt file to compile to plain javascript. This phase is probably not completed when it's imported by npm (or apm).

Hope it helps !

**Comment by Glavin001** [comment]
Found the problem! https://github.com/Glavin001/Coffee-Formatter/blob/master/package.json#L32
It points to the wrong repo URL and also does not properly build from the litcoffee into JS.

**Comment by Glavin001** [comment]
Published to v0.18.0. Happy New Year! :smiley:

**Comment by alkama** [comment]
Thanks :)
Happy new year !
Works fine now! (thought the plug does strange things that should probably be issues on the coffee-formatter itself).
